pkg/build/options.go

```diff
@@ -116,6 +116,24 @@ func withBuilder(b builder) Option {
 	}
 }
 
+// WithGoVersionSBOM is a functional option to direct ko to use
+// go version -m for SBOM format.
+func WithGoVersionSBOM() Option {
```

Comment:
I'll probably make this `WithSBOM(impl)` but that can happen later.

Reply:
I hate the signature for `sbomber`, so I didn't go that route (I started to), but I'd love to head that direction if you have ideas.